frontend/simple/components/CreateGroup/CreateGroupRules.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div id="rulesStep" class="has-text-centered">
+    <h1 class="title is-2"><i18n>Group Advanced Rules</i18n></h1>
+    <p class="content"><i18n>What percentage approval is necessary to adjust the group rules?</i18n></p>
+    <div class="columns">
+      <div class="column">
+        <div class="rule">
+          <p class="percent">{{ group.changePercentage }}%</p>
+          <circle-slider
+            class="circle-slider"
+            :value="group.changePercentage"
+            @input="value => update('changePercentage', value)"
+            :side="160"
+            :circleWidth="1"
+            :progressWidth="2"
+            :knobRadius="5"
+            :progressColor="changeColor"
+            :knobColor="changeColor"
+            circleColor="#69707a"
+          >
+          </circle-slider>
+          <p class="title is-6"><i18n>Change Rules</i18n></p>
+        </div>
+      </div>
+      <div class="column">
+        <div class="rule">
+          <p class="percent">{{ group.memberApprovalPercentage }}%</p>
+          <circle-slider
+            class="circle-slider"
+            :value="group.memberApprovalPercentage"
+            @input="value => update('memberApprovalPercentage', value)"
+            :side="160"
+            :circleWidth="1"
+            :progressWidth="2"
+            :knobRadius="5"
+            :progressColor="approveColor"
+            :knobColor="approveColor"
+            circleColor="#69707a"
+          >
+          </circle-slider>
+          <p class="title is-6"><i18n>Add Member</i18n></p>
+        </div>
+      </div>
+      <div class="column">
+        <div class="rule">
+          <p class="percent">{{ group.memberRemovalPercentage }}%</p>
+          <circle-slider
+            class="circle-slider"
+            :value="group.memberRemovalPercentage"
+            @input="value => update('memberRemovalPercentage', value)"
+            :side="160"
+            :circleWidth="1"
+            :progressWidth="2"
+            :knobRadius="5"
+            :progressColor="removeColor"
+            :knobColor="removeColor"
+            circleColor="#69707a"
+          >
+          </circle-slider>
+          <p class="title is-6"><i18n>Remove Member</i18n></p>
+        </div>
+      </div>
+    </div>
+    <transition name="open">
+      <div class="message is-warning" v-if="!superMajority">
+        <div class="message-body">
+          The percentage value you are choosing is most likely too low
+          for a decision that can have a potentially significant impact
+          on a person’s life. Please consider using a supermajority threshold.
+          <a href="https://groupincome.org/2016/09/deprecating-mays-theorem/#when-majority-rule-can-harm">
+            Read more on our blog about the dangers of majority rule.
+          </a>
```

Comment:
Nice, but remember to put all text, including this, into `<i18n>` tags. ^_^

Reply:
shoot!